.flake8

```diff
@@ -1,4 +1,9 @@
 [flake8]
-max-line-length = 88
+max-line-length = 127
```

Comment:
Could you revert this back to 88 please?

Reply:
Thank you, I will revert.